src/GoodsList.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
 import { Good } from './types/Good';
 
 type Props = {
   goods: Good[]
 };
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
+  <List>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <ListItem
+        key={good.id}
+        data-cy="good"
+        style={{ color: good.color }}
```

Comment:
let's create class names for colors and add class names from `good.color`

Reply:
fixed